assets/multiqc_config.yml

```diff
@@ -3,9 +3,9 @@ custom_logo_url: https://github.com/nf-core/sarek/
 custom_logo_title: "nf-core/sarek"
 
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/sarek/releases/tag/3.3.0" target="_blank">nf-core/sarek</a>
+  This report has been generated by the <a href="https://github.com/nf-core/sarek/releases/tag/dev" target="_blank">nf-core/sarek</a>
```

Comment:
Note this says dev, I assume that this is template update is just going into the dev branch and this string will be updated later.

Reply:
that's the plan